ocaml/xapi/xapi_pool.ml

```diff
@@ -1062,7 +1062,7 @@ let eject ~__context ~host =
     let write_first_boot_management_interface_configuration_file () =
       (* During pool.eject firstboot scripts running on the ejected host will write the new
```

Comment:
Please update this comment to say that the inventory now has an empty MANAGEMENT_INTERFACE at first boot.